src/pages/List/List.js

```diff
@@ -109,7 +119,7 @@ function List() {
 
   return (
     <ListContent>
-      <Result />
+      <Result category_txt={location.state.category_txt} />
```

Comment:
- 여기도 category_txt 말고 카멜 케이스로 통일해 주세요!  
- 그리고 txt 로 줄여쓰지 마시고, 풀네임 사용해주세요!
```suggestion
      <Result categoryText={location.state.category_txt} />
```